### ISSUE TITLE
Images - Missing alt text

### ISSUE THREAD

**Issue by bradystroud** [SSWConsulting/SSW.Rules.Content#345]
cc: @JackLeerson @adamcogan 
Hi @tiagov8,  

CodeAuditor is finding some errors in the HTML where images do not have alt text. [https://htmlhint.com/docs/user-guide/rules/alt-require](https://htmlhint.com/docs/user-guide/rules/alt-require)

You can see all the rules with images missing alt text [here.](https://codeauditor.com/htmlhint/7f5d5ca9-a5e3-4823-bb5a-b050dd811fc8?displayMode=1)

![2021-03-22_14-38-02](https://user-images.githubusercontent.com/38869720/111941203-3d605280-8b1c-11eb-91ed-571224879d4d.png)
**Figure: Image missing alt text**

The alt text can be easily added in the Markdown using this syntax:
```md
![Figure: Good figure](https://images.unsplash.com/photo-1491472253230-a044054ca35f)
```

1. Please go through all the rules and add alt text (figure statements)

Thanks,

-Brady

**Comment by tiagov8** [comment]
Hey Brady,

It seems like the very first issue has an issue 🤓 :
<img width="890" alt="Screen Shot 2021-03-23 at 5 09 37 PM" src="https://user-images.githubusercontent.com/12601228/112234868-b1712800-8bfa-11eb-9d69-beeba1955848.png">

<img width="1297" alt="Screen Shot 2021-03-23 at 5 13 05 PM" src="https://user-images.githubusercontent.com/12601228/112235024-0dd44780-8bfb-11eb-95a2-99e292ff99ff.png">

See how the syntax seems alright:

https://www.ssw.com.au/rules/login-security-do-you-know-the-correct-error-message-for-an-incorrect-user-name-or-password
<img width="1012" alt="Screen Shot 2021-03-23 at 5 09 20 PM" src="https://user-images.githubusercontent.com/12601228/112234840-9c949480-8bfa-11eb-84f8-2aa36e094e11.png">

1. Please investigate why this Markdown was not converted to an "alt"
2. Make sure it's fixed and give me a new CA report to fix
3. I noticed we have some images in rules where we don't want captions... how would we have them if we need this syntax?

Cheers,
Tiago
